src/Kunstmaan/UserManagementBundle/Resources/views/Users/edit.html.twig

```diff
@@ -45,8 +45,14 @@
         </div>
 
         <!-- Fields -->
+
+
         <fieldset class="form__fieldset--padded">
-            {{ form_rest(form) }}
+            {% if tabPane is defined %}
+                {{ tabs_widget(tabPane) }}
+            {% else %}
```

Comment:
I don't think the else is needed here, but we should always have a form_end(form)